tbapy/main.py

```diff
@@ -358,19 +358,18 @@ def district_rankings(self, district):
         """
         return [DistrictRanking(raw) for raw in self._get('district/%s/rankings' % district)]
 
-    def district_teams(self, district, year, keys=False):
+    def district_teams(self, district, keys=False):
         """
         Get list of teams in the given district in a certain year.
 
         :param district: Key for the district to get teams in.
-        :param year: Year from which to get teams.
         :param keys: Return list of team keys only rather than full data on every team.
         :return: List of string keys or Team objects.
         """
         if keys:
-            return self._get('district/%s/%s/teams/keys' % (district, year))
+            return self._get('district/{district_key}/teams/keys'.format(district_key=district))
```

Comment:
While this is proper formatting for 3.x, it’s probably best not to change it unless you’re gonna change it everywhere

Reply:
Yeah, I admit it looks out of place.